tools/jarcat/tar/tar_test.go

```diff
@@ -0,0 +1,105 @@
+package tar
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"io"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const testDataDir = "tools/jarcat/tar/test_data"
+
+func TestNoCompression(t *testing.T) {
+	filename := "test_no_compression.tar"
+	err := Write(filename, testDataDir, "", false)
```

Comment:
should this and the other unit tests defer-call a best-effort deletion of the output file?